src/components/Certify/Certify.stories.ts

```diff
@@ -21,5 +21,6 @@ Default.args = {
   isCertified: false,
   message: 'Note: It is an offence to make a false or misleading statement in respect of a material fact in a \n' +
   'record submitted to the Corporate Registry for filing. See section 427 of the Business Corporations Act.',
-  entityDisplay: 'BC Company'
+  entityDisplay: 'BC Company',
+  disableEdit: false
```

Comment:
Is it possible to add a small unit test or 2 to validate this new prop?

Reply:
100%! Will do 👍

Reply:
Updated!